### DISCUSSION TITLE
3.24.0 图片视图还是不能显示文件夹

### DISCUSSION THREAD

**Discussion by li-peifeng** [AlistGo/alist#4886] [General]
看alist-web版本介绍里说此版本能在图片模式下显示文件夹，可我升级后有全局设置，可是并不能显示文件夹，有升级的朋友确认下此功能正常吗？

**Comment by anwen-anyi** [comment]
无任何问题啊 你得自己配置吖
![image](https://github.com/alist-org/alist/assets/56105412/2fdae52b-a12a-47b5-9f59-9f6a961d3cb5)

**Reply by li-peifeng** [reply]
![image](https://github.com/alist-org/alist/assets/103113557/90cf0481-51e9-4fc4-9d39-0c3f06409aca)

![image](https://github.com/alist-org/alist/assets/103113557/491c0d88-39c9-400c-8fa5-7726393561f7)

**Comment by li-peifeng** [comment]
> 无任何问题啊 你得自己配置吖
> 
> ![image](https://github.com/alist-org/alist/assets/56105412/2fdae52b-a12a-47b5-9f59-9f6a961d3cb5)
> 
> 

已经设置了，重新下载了最新的docker版本，清除了缓存还是一样，选了图片模式就是空白的

**Comment by xhofe** [comment]
当前的判定有些问题 有图像时才会显示。

下个版本修复。

**Reply by li-peifeng** [reply]
哦，好的，谢谢！
